common/pulp_docker/common/constants.py

```diff
@@ -67,3 +68,7 @@
 # on the repo scratchpad
 IMAGE_TAG_KEY = 'tag'
 IMAGE_ID_KEY = 'image_id'
+
+MEDIATYPE_MANIFEST_LIST = 'application/vnd.docker.distribution.manifest.list.v2+json'
+MEDIATYPE_MANIFEST_S1 = 'application/vnd.docker.distribution.manifest.v1+json'
```

Comment:
theoretically there might be a signed manifest of schema 1 which would have media type "application/vnd.docker.distribution.manifest.v1+prettyjws”

Reply:
Is "application/vnd.docker.distribution.manifest.v1+prettyjws” a valid media type for an Image Manifest in the Manifest List?

Reply:
image manifest, it is a signed image manifest schema1.